ceph/ceph_admin.py

```diff
@@ -548,8 +548,55 @@ def ceph_iscsi(self, nodes, **kwargs):
             nodes: iscsi node list
             kwargs: key-value arguments
```

Comment:
Please document what is supported and what are the required keys. If it is for future use, please document it. IMO, future use... can be avoided.

Reply:
yeah its been for future-use.